test/tests/problems/eigen_problem/initial_condition/gold/ne_ic_out_inverse_eigenvalues_0001.csv

```diff
@@ -1,2 +1,2 @@
-eigen_values_real,eigen_values_imag
-5.0014600981294,0
+eigen_values_imag,eigen_values_real
+0,5.0014600976647
```

Comment:
In my head, I always want to see "real" first, and then "imag"?

What motivate you to change this?

Reply:
No, it is automatically generated. I do not know why the two are switched.

Reply:
I am guessing moose is ordering alphabetically. I do not like this either, but that is the csv file.

Reply:
I was worried about there is a bug. I could help you on this if you do not have time.

Reply:
This could be due to a recent reporter change? I think this is fine.

Reply:
It is super weird. We declared real first

Reply:
@zachmprince or @aeslaughter Do you know why we are having this change now in this CSV file? we prefer to see real first for a complex number but the code put imag before real even in the VPP real is declared first.

Reply:
Looks like the list of vpp_name/vec_name is gathered as a `std::set` in TableOutput. So they are automatically being sorted. I'm not sure what was there before, but this is why you are seeing imag first.

Reply:
I'm curious on why this file needs to be regolded. CSVDiff doesn't care about the ordering of the columns, does it?

Reply:
no it does not. I happened to deleted this previously and add it back with my latest generated file. Then we see this ;-)

Reply:
OK. I see